test/evergreen.yml

```diff
@@ -1614,6 +1621,25 @@ tasks:
               cp -rf WT_TEST WT_TEST_$file
             done
 
+  - name: time-shift-sensitivity-test
+    depends_on:
+      - name: compile-enable-strict
+    commands:
+      - func: "fetch artifacts"
+        vars:
+          dependent_task: compile-enable-strict
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix/test/csuite"
+          script: |
+            set -o errexit
+            set -o verbose
+
+            rundir=../../../test/csuite
+            cp $rundir/time_shift_test.sh ./
+            chmod a+x ./time_shift_test.sh
+            ./time_shift_test.sh /usr/local/lib/faketime/libfaketimeMT.so.1 0-1
```

Comment:
PRE_LOAD /usr/local/lib/faketime/libfaketimeMT.so.1 in the test_env_vars. 
Can we get rid of these?
rundir=../../../test/csuite
cp $rundir/time_shift_test.sh ./
chmod a+x ./time_shift_test.sh